client/v3/logger.go

```diff
@@ -51,18 +51,28 @@ func etcdClientDebugLevel() zapcore.Level {
 		return zapcore.InfoLevel
 	}
 	var l zapcore.Level
-	if err := l.Set(envLevel); err == nil {
+	if err := l.Set(envLevel); err != nil {
 		log.Printf("Deprecated env ETCD_CLIENT_DEBUG value. Using default level: 'info'")
 		return zapcore.InfoLevel
 	}
 	return l
 }
 
+func etcdClientLogFormat() string {
+	envFormat := os.Getenv("ETCD_CLIENT_LOG_FORMAT")
```

Comment:
Let's document this in the https://github.com/etcd-io/etcd/blob/e2d67f2e3bfa6f72178e26557bb22cc1482c418c/client/v3/doc.go#L102-L104

as well add a changelog entry similar to how the ETCD_CLIENT_DEBUG was added.